### PR TITLE
Create Tabs component in UI kit

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
- Add Tabs component to UI kit according to updated design

https://github.com/user-attachments/assets/a705f3ae-5764-4327-aa1d-03a1bd0e6a13

## Justify why they are needed
New design and better reuse 

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
* **#4668** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4668?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gustaveen and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/ui/src/components/Tabs/Tabs.css.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { createVar, style, styleVariants } from '@vanilla-extract/css'
+import { xStack } from '../../patterns'
+import { tokens } from '../../theme'
+
+const ButtonActiveColor = createVar()
+
+export const tabsRoot = style({
+  paddingInline: tokens.space.xs,
+  marginInline: 'auto',
+  width: 'min-content',
```
Comment (gustaveen) [inline packages/ui/src/components/Tabs/Tabs.css.ts:10]: Make sure tabs are not wider than it's content.

File: packages/ui/src/components/Tabs/Tabs.css.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+export const tabsListSize = styleVariants({
+  small: { borderRadius: tokens.radius.xs },
+  medium: { borderRadius: tokens.radius.md },
+  large: { borderRadius: tokens.radius.lg },
+})
+
+export const tabsListType = styleVariants({
+  default: {
+    vars: { [ButtonActiveColor]: tokens.colors.buttonSecondary },
```
Comment (gustaveen) [inline packages/ui/src/components/Tabs/Tabs.css.ts:25]: Set button color based on tab `type`

File: packages/ui/src/theme/colors/colors.ts [hunk 3]
```diff
@@ -334,6 +334,10 @@ export const colors = {
   offWhite: gray[25],
   offWhiteTranslucent: grayTranslucent[25],
 
+  // Surface
```
Comment (gustaveen) [inline packages/ui/src/theme/colors/colors.ts:337]: New colors from UI kit

**Review by alebedev — APPROVED** [review]
Great work!

**Comment by gustaveen** [comment]
### Merge activity

* **Sep 3, 7:55 AM EDT**: @gustaveen started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4668).

**Pull request merged**